tiles-generation/docker/vector-tile/modified-s5cmd/s3.go

```diff
@@ -0,0 +1,969 @@
+//  Copyright OpenSearch Contributors
+//  SPDX-License-Identifier: Apache-2.0
+
+// In order to support add metadata 'Content-Encoding=gzip' and 'Content-Type=application/x-protobuf' on copy command,
```

Comment:
Did you try extending from library instead of copying complete src code?

Reply:
Offline discussed with Vijay, it can't be used by extending from library due to method not public. And I will keep the existing method that replacing some codebase and build them in docker. I will also create a github issue on s5cmd repo to support the feature.